lib/src/summary.dart

```diff
@@ -156,21 +156,11 @@ class Suggestion extends Object
   Suggestion(this.level, this.title, this.description, {this.file});
 
   factory Suggestion.bug(Object error, StackTrace stack) {
-    var lines = LineSplitter
-        .split([
-          'File an issue at `https://github.com/dart-lang/pana/`',
-          'Include the version of `pana` and any other details.',
-          error,
-          '',
-          stack
-        ].join('\n'))
-        .take(100)
-        .join('\n');
-
-    return new Suggestion(
-        SuggestionLevel.bug,
-        'There is a bug in the `pana` package or one of its dependencies.',
-        lines);
+    var lines =
```

Comment:
Why remote the "File an issue..." part?

Reply:
Because context matters. If the package site hits a bug, it should print its own `report an issue` link, ideally with a pre-filled body w/ the pana version, etc.

Don't want to sprinkle links all over